networkx/algorithms/centrality/harmonic.py

```diff
@@ -13,7 +13,7 @@
 __all__ = ['harmonic_centrality']
 
 
-def harmonic_centrality(G, u=None, distance=None):
```

Comment:
The option to compute the harmonic centrality for a single node was indeed a good choice because the user can limit the computation to a set of nodes.   removing this option force the computation on the entire graph, which requires more time. 

I strongly recommend to roll back to the implementation that included the single-node computation!